models/staging/stg_amazon_selling_partner__order_item.sql

```diff
@@ -0,0 +1,85 @@
+{{ config(enabled=var('amazon_selling_partner__using_orders_module', true)) }}
+
+{% set base_table = ref('stg_amazon_selling_partner__order_item_base') if var('amazon_selling_partner_sources',[]) != [] else source('amazon_selling_partner', 'order_item') %}
+
+with base as (
+
+    select * 
+    from {{ base_table }}
+),
+
+fields as (
+
+    select
+        {{
+            fivetran_utils.fill_staging_columns(
+                source_columns=adapter.get_columns_in_relation(base_table),
+                staging_columns=get_order_item_columns()
+            )
+        }}
+        
+        {{ amazon_selling_partner_apply_source_relation() }}
+        
+    from base
+),
+
+final as (
+    
+    select 
+        source_relation, 
+        amazon_order_id,
+        order_item_id,
+        asin,
+        seller_sku,
+        title,
+        product_info_detail_number_of_items,
+        cast({{ dbt.date_trunc('day', 'scheduled_delivery_start_date') }} as date) as scheduled_delivery_start_date,
+        cast({{ dbt.date_trunc('day', 'scheduled_delivery_end_date') }} as date) as scheduled_delivery_end_date,
+        quantity_ordered,
+        quantity_shipped,
+
+        item_price_amount,
+        item_price_currency_code,
+        item_tax_amount,
+        item_tax_currency_code,
+        shipping_discount_amount,
+        shipping_discount_currency_code,
+        shipping_discount_tax_amount,
+        shipping_discount_tax_currency_code,
+        shipping_price_amount,
+        shipping_price_currency_code,
+        shipping_tax_amount,
+        shipping_tax_currency_code,
+        promotion_discount_amount,
+        promotion_discount_currency_code,
+        promotion_discount_tax_amount,
+        promotion_discount_tax_currency_code,
+
+        condition_id,
+        condition_note,
+        condition_subtype_id,
+        buyer_requested_cancel_buyer_cancel_reason as buyer_requested_cancel_reason,
+        buyer_requested_cancel_is_buyer_requested_cancel as is_buyer_requested_cancel,
+        deemed_reseller_category,
+        ioss_number,
+        is_gift,
+        is_transparency,
+        serial_number_required as is_serial_number_required, -- only populated for Easy Ship orders
+        store_chain_store_id,
+        tax_collection_model, -- always MarketplaceFacilitator in US
+        tax_collection_responsible_party -- always Amazon Web Services in US
+
+        {# columns i'm excluding -- remove later #}
```

Comment:
Reminder to remove if still planning to exclude

Reply:
removed